### DISCUSSION TITLE
[Hotfix] Fixes for macOS Emulation and more.

### DISCUSSION THREAD

**Discussion by cryptofyre** [ciderapp/Apple-Music-Electron#163] [General]
# Hotfix
This update provides a hotfix which allows the stoplights in macOS Emulation mode to function again as Apple pushed an update that broke it.

# Changelog:
someone write this because pain

Apple stop torturing us with stupid Content Security Policies and CORS requests.
[balls](https://cryptofyre.org/cdn/bruhlmfao.mp4)

<hr /><em>This discussion was created from the release <a href='https://github.com/Apple-Music-Electron/Apple-Music-Electron/releases/tag/v2.4.1'>[Hotfix] Fixes for macOS Emulation and more.</a>.</em>

**Comment by alfaaarex** [comment]
Still the custom titlebar isn't working.

**Comment by dsancheztrujillo** [comment]
How can I install it ? Im from Perú. Every time that I try to log in appears this message

![image](https://user-images.githubusercontent.com/68398565/129421502-8964374e-9292-4ca2-8fce-e8128bc1bc83.png)

**Reply by coredev-uk** [reply]
https://github.com/Apple-Music-Electron/Apple-Music-Electron/wiki/Troubleshooting#authentication-issues
